releasenotes/notes/sparsepauliop-from-index-list-4660fdaa492cd8b2.yaml

```diff
@@ -0,0 +1,19 @@
+---
+features:
+  - |
+    The :meth:`.SparsePauliOp.from_list` constructor can now take an iterable where
+    the elements represent Pauli terms that are themselves sparse, so that
+    `"XIIIIIIIIIIIIIIIIX"` can now be written as `("XX", [0, 16])`. For example, the operator
+
+    .. math::
+
+      H = X_0 Z_3 + 2 Y_1 Y_4
+
+    can now be constructed as
+
+    .. code-block:: python
+
+      op = SparsePauliOp.from_list([("XZ", [0, 3], 1), ("YY", [1, 4], 2)], num_qubits=5)
+
+    This greatly facilitates the construction of very sparse operators on many qubits, as
+    is often the case for Ising Hamiltonians.
```

Comment:
The release note needs updating to account for the splitting out of `from_sparse_list` - `from_list` itself remains unchanged, right?